sass/cd/_cd-variables.scss

```diff
@@ -120,9 +120,23 @@ $cd-primary--dark: $cd-blue--dark;
 $cd-font: helvetica, arial, sans-serif;
 $cd-font--system: system-ui;
 
-// Web fonts.
-//@import url('https://fonts.googleapis.com/css?family=Mali:300,400,500,600,700&display=swap"');
-//$cd-font--webfont: 'Mali', cursive;
+// sass-lint:disable no-url-domains no-url-protocols
+
+// Multilingual
+// https://brand.unocha.org/d/xEPytAUjC3sH/visual-identity#/basics/fonts/multi-language-typefaces-1
+@import url('//fonts.googleapis.com/css2?family=Noto+Sans:wght@400;700&display=swap');
+$cd-font--noto: 'Noto Sans', sans-serif;
```

Comment:
Starting a new thread about Noto Sans in particular since the line containing the URL was the basis for the self-hosted versus CDN discussion.

This particular Noto font is "the Sans Latin, Greek and Cyrillic family." — Arabic and Chinese are not included in this font.

For DSR, we were advised by DDMU to use [Noto Kufi for Arabic](https://www.google.com/get/noto/#kufi-arab) for titles, and Dubai for body copy, both of which are **not** available on CDNs. You can see my [declarations on DSR here](https://github.com/UN-OCHA/reports-site/blob/dev/layouts/default.vue#L249-L271).

For GHO we used Kufi for all text, and did not use Dubai. Coincidentally, their Arabic page is live as of today so you can inspect here: https://gho.unocha.org/ar

I have personally never installed Chinese-compatible fonts so I'd probably start by asking DDMU what their go-to font is before we pick one of the 10 or so options available on the Noto site.

Reply:
AFAIK performance-wise the browsers have most of this stuff figured out, as long as we aggressively scope our selectors defining font usage with `[lang='XX']` prefixes. That is, the fonts will not be downloaded until the CSSOM matches an actual DOM element worth triggering said download.

Reply:
@rupl Great, thanks for elaborating. I will follow up.

Reply:
For foreign char sets we once used this

```css
/*Chinese*/

/*Traditional*/
@font-face {
  font-family: 'NotoChineseTraditionalRegular';
  font-display: swap;
  src: url('/fonts/NotoSansCJKtc-Regular-Alphabetic.woff2') format('woff2'), url('/fonts/NotoSansCJKtc-Regular-Alphabetic.woff') format('woff'), url('/fonts/NotoSansCJKtc-Regular-Alphabetic.ttf') format('truetype');
}

@font-face {
  font-family: 'NotoChineseTraditionalBold';
  font-display: swap;
  src: url('/fonts/NotoSansCJKtc-Bold-Alphabetic.woff2') format('woff2'), url('/fonts/NotoSansCJKtc-Bold-Alphabetic.woff') format('woff'), url('/fonts/NotoSansCJKtc-Bold-Alphabetic.ttf') format('truetype');
}

/*Simplified*/
@font-face {
  font-family: 'NotoChineseSimplifiedRegular';
  font-display: swap;
  src: url('/fonts/NotoSansCJKsc-Regular-Alphabetic.woff2') format('woff2'), url('/fonts/NotoSansCJKsc-Regular-Alphabetic.woff') format('woff'), url('/fonts/NotoSansCJKsc-Regular-Alphabetic.ttf') format('truetype');
}

@font-face {
  font-family: 'NotoChineseSimplifiedBold';
  font-display: swap;
  src: url('/fonts/NotoSansCJKsc-Bold-Alphabetic.woff2') format('woff2'), url('/fonts/NotoSansCJKsc-Bold-Alphabetic.woff') format('woff'), url('/fonts/NotoSansCJKsc-Bold-Alphabetic.ttf') format('truetype');
}

/*Japanese*/
@font-face {
  font-family: 'NotoJapanRegular';
  font-display: swap;
  src: url('/fonts/NotoSansCJKjp-Regular-Alphabetic.woff2') format('woff2'), url('/fonts/NotoSansCJKjp-Regular-Alphabetic.woff') format('woff'), url('/fonts/NotoSansCJKjp-Regular-Alphabetic.ttf') format('truetype');
}

@font-face {
  font-family: 'NotoJapanBold';
  font-display: swap;
  src: url('/fonts/NotoSansCJKjp-Bold-Alphabetic.woff2') format('woff2'), url('/fonts/NotoSansCJKjp-Bold-Alphabetic.woff') format('woff'), url('/fonts/NotoSansCJKjp-Bold-Alphabetic.ttf') format('truetype');
}

/*Greek*/
@font-face {
  font-family: 'NotoGreekRegular';
  font-display: swap;
  src: url('/fonts/NotoGreekRegular.woff2') format('woff2'), url('/fonts/NotoGreekRegular.woff') format('woff'), url('/fonts/NotoGreekRegular.ttf') format('truetype');
}

@font-face {
  font-family: 'NotoGreekBold';
  font-display: swap;
  src: url('/fonts/NotoGreekBold.woff2') format('woff2'), url('/fonts/NotoGreekBold.woff') format('woff'), url('/fonts/NotoGreekBold.ttf') format('truetype');
}

/*Russia*/
@font-face {
  font-family: 'NotoRussianRegular';
  font-display: swap;
  src: url('/fonts/NotoRussianRegular.woff2') format('woff2'), url('/fonts/NotoRussianRegular.woff') format('woff'), url('/fonts/NotoRussianRegular.ttf') format('truetype');
}

@font-face {
  font-family: 'NotoRussianBold';
  font-display: swap;
  src: url('/fonts/NotoRussianBold.woff2') format('woff2'), url('/fonts/NotoRussianBold.woff') format('woff'), url('/fonts/NotoRussianBold.ttf') format('truetype');
}
```

Reply:
good to know!